uber/templates/preregistration/noise_levels.html

```diff
@@ -1 +0,0 @@
-{% comment %}This section to be overridden only by Magstock, not used by anything else{% endcomment %}
```

Comment:
This was left over from last year.  I think Dom added it when he was doing the MAGStock site last time around.  (It definitely wasn't me, since my text editor always a newline at the end of every file, and this one doesn't end with a newline.)